evennia/commands/default/muxcommand.py

```diff
@@ -79,6 +79,11 @@ def parse(self):
         it here). The rest of the command is stored in self.args, which can
         start with the switch indicator /.
 
+        Optional variables to aid in parsing, if set:
+          self.options = (tuple of valid /switches expected
+                          by this command (without the /))
+          self.split = Alternate string delimiter to separate left/right hand side.
```

Comment:
Is there no better name for the new optional `MuxCommand` class variables than what you've requested?
```
        Optional variables to aid in parsing, if set:
          self.parser_switch_options  - (tuple of valid /switches expected by this
                                         command (without the /))
          self.parser_rhs_split       - Alternate string delimiter to separate
                                        left/right hand side.
```
This requires updating all explicit settings of them when creating a command based on this command class:
```
    key = "@tel"
    aliases = "@teleport"
    parser_switch_options  = ("quiet", "intoexit", "tonone", "loc")
    parser_rhs_split = "to"
    locks = "cmd:perm(teleport) or perm(Builder)"
    help_category = "Building"
```
which appears out of place among the others that are not appended with `func_`.

Would `switch_options` and `rhs_split` be a good compromise, or perhaps something else?

Reply:
The others wouldn't be prepended with `func` since they apply to the entire Command class, not just to `func`. These switches are the first to apply only to a single sub-method. But for the sake of  brevity I think `switch_options` and `rhs_split` are ok too.